src/pyskindose/plotting/create_dose_map_plot.py

```diff
@@ -1,21 +1,256 @@
         notebook_mode=settings.plot.notebook_mode)
+
+    PLOT_MARGINS = fetch_plot_margin(notebook_mode=settings.plot.notebook_mode)
+
+    lat_text = [f"<b>lat:</b> {np.around(patient.r[ind, 2],2)} cm<br>"
+                for ind in range(len(patient.r))]
+
+    lon_text = [f"<b>lon:</b> {np.around(patient.r[ind, 0],2)} cm<br>"
+                for ind in range(len(patient.r))]
+
+    ver_text = [f"<b>ver:</b> {np.around(patient.r[ind, 1],2)} cm<br>"
+                for ind in range(len(patient.r))]
+
+    dose_text = [f"<b>skin dose:</b> {round(patient.dose[ind],2)} mGy"
+                 for ind in range(len(patient.r))]
+
+    hover_text = [lat_text[cell] + lon_text[cell] + ver_text[cell] +
+                  dose_text[cell] for cell in range(len(patient.r))]
+
+    # create mesh object for the phantom
+    phantom_mesh = [
+        go.Mesh3d(
+            x=patient.r[:, 0], y=patient.r[:, 1], z=patient.r[:, 2],
+            i=patient.ijk[:, 0], j=patient.ijk[:, 1], k=patient.ijk[:, 2],
+            intensity=patient.dose, colorscale=DOSEMAP_COLORSCALE,
+            showscale=True,
+            hoverinfo='text',
+            text=hover_text, name="Human",
+            colorbar=dict(tickfont=dict(color=COLOR_PLOT_TEXT),
+                          title="Skin dose [mGy]",
+                          titlefont=dict(
+                            family=PLOT_FONT_FAMILY,
+                            color=COLOR_PLOT_TEXT)))]
+
+    # Layout settings
+    layout = go.Layout(
+        height=PLOT_HEIGHT,
+        width=PLOT_WIDTH,
+        margin=PLOT_MARGINS,
+
+        font=dict(
+            family=PLOT_FONT_FAMILY,
+            color=COLOR_PLOT_TEXT,
+            size=PLOT_FONT_SIZE),
+
+        hoverlabel=dict(
+            font=dict(
+                family=PLOT_HOVERLABEL_FONT_FAMILY,
+                size=PLOT_HOVERLABEL_FONT_SIZE)),
+
+        titlefont=dict(
+            family=PLOT_FONT_FAMILY,
+            size=PLOT_FONT_SIZE,
+            color=COLOR_PLOT_TEXT),
+
+        paper_bgcolor=COLOR_CANVAS,
+
+        scene=dict(
+            aspectmode=PLOT_ASPECTMODE_PLOT_DOSEMAP,
+            xaxis=dict(
+                title='',
+                backgroundcolor=COLOR_CANVAS,
+                showgrid=False,
+                zeroline=False,
+                showticklabels=False,
+                ),
+
+            yaxis=dict(
+                title='',
+                backgroundcolor=COLOR_CANVAS,
+                showgrid=False,
+                zeroline=False,
+                showticklabels=False,
+                ),
+
+            zaxis=dict(
+                title='',
+                backgroundcolor=COLOR_CANVAS,
+                showgrid=False,
+                zeroline=False,
+                showticklabels=False,
+                ),
+                )
+        )
+
+    # create figure
+    fig = go.Figure(data=phantom_mesh, layout=layout)
+
+    if settings.plot.interactivity:
+        fig.show()
+
+    else:
```

Comment:
If you add a `return` after `fig.show()` above you can skip this else and reduce the nesting

Reply:
Resolved in c27f2095733acffaee2458dee282612628288355